core/chains/evm/config/v2/config.go

```diff
@@ -150,6 +150,11 @@ func (cs EVMConfigs) Nodes() (ns []types.Node) {
 			ns = append(ns, legacyNode(n, cs[i].ChainID))
 		}
 	}
+	//Add unique ID to prevent overwriting nodes during unmarshalling since this will be transformed into JAID
+	for i := range ns {
+		ns[i].ID = int32(i)
+	}
+
```

Comment:
Why do they need an ID at all? Name should be the unique identifier

Reply:
Yeah we should be able to make that change now that the DB variant is gone, but it would require multiple PRs across repos, so this fixes things for now.

Reply:
Yeah if that is easy to do then it seems fitting :+1: